charts/cron-tfjob/templates/_tfjob.yaml

```diff
@@ -29,6 +29,15 @@ spec:
 {{- if eq "Running" $cleanPodPolicy }}
   cleanPodPolicy: Running
 {{- end }}
+{{- end }}
+{{- if .Values.tfjob.activeDeadlineSeconds }}
+  activeDeadlineSeconds: {{ .Values.tfjob.activeDeadlineSeconds }}
+{{- end }}
```

Comment:
Please update changelog and chart version.

Reply:
Done